xarray/tests/test_backends.py

```diff
@@ -3621,11 +3621,15 @@ def create_tmp_geotiff(
     ny=3,
     nz=3,
     transform=None,
-    transform_args=[5000, 80000, 1000, 2000.0],
-    crs={"units": "m", "no_defs": True, "ellps": "WGS84", "proj": "utm", "zone": 18},
+    transform_args=None,
+    crs=None,
```

Comment:
unfortunately, since the default values for parameters to `rasterio.open` are mostly `None` we can't use it as default value: otherwise, how would we explicitly request `rasterio.open`'s default?

We might fix this by using a different value, e.g. a `object` instance:
```python
_default_value = object()
```

Reply:
you mean by replacing None with object() should fix this?

Reply:
@keewis Could you please take a look at other values also and suggest changes?

Reply:
no, what I meant was that we define `default_value` somewhere outside the function (at module level):
```python
default_value = object()
```
and then use that everywhere you have been using `None`:
```python
    transform_args=default_value,
    crs=default_value,
```
and later:
```python
if transform_args is default_value:
    transform_args = [...]

if crs is default_value:
    crs = {...}
```

the parameters of `roundtrip` are only used as kwargs, so they're fine